parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -513,8 +523,8 @@ returnRelease() {
     local osVersionWithoutDot=$(echo "${osVersion}" | sed 's/\.//g')
     #For UBUNTU, if $osVersion is 18.04 and "r1804" is also defined in components.json, then $release is set to "r1804"
     #Similarly for 20.04 and 22.04. Otherwise $release is set to .current.
-    #For MARINER, the release is always set to "current" now.
-    if [[ "${os}" == "${MARINER_KATA_OS_NAME}" || "${os}" == "${MARINER_OS_NAME}" ]]; then
+    #For MARINER/AZURELINUX, the release is always set to "current" now.
+    if [[ "${os}" == "${MARINER_KATA_OS_NAME}" ]] || isMarinerOrAzureLinux "${os}"; then
```

Comment:
does Mariner kata os also not fall under the same configuration criteria as mariner/azure linux? can we add that to the isMarinerOrAzureLinux function?

Reply:
Checking for MARINER_KATA_OS_NAME in the isMarinerOrAzureLinux function - [commit](https://github.com/Azure/AgentBaker/pull/4808/commits/e47defbfc47950f37717e3ac26506df559386026)

Running the test vhd pipeline - https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=101853413&view=results